src/components/subscribemodal/index.tsx

```diff
@@ -43,6 +45,32 @@ const SubscribedModal: React.FC<SubscribedModalProps> = ({ onClose }) => {
   );
 };
 
+const CloseButton = styled.div`
```

Comment:
i would suggest create 1 styled component called `CloseButton` and put all the styles in there, no need to create extra JSX. 
```
const CloseButton = styled.a``
```